File: angular-proj/src/app/DTO/Booking.ts
export class Booking {
    BookingID: number;
    BookingName: string;
    BookingAmount: number;
    BookingStatus: number;
    BookingTimeStamp: Date;
    BookingSeats: Array<number>;

    constructor(){
        this.BookingID = 0;
        this.BookingName = "";
        this.BookingAmount = 0.00;
        this.BookingStatus = 0;
        this.BookingTimeStamp = new Date();
        this.BookingSeats = new Array<number>();
    }
}

File: api-project/MovieReservation/MovieReservation/DTO/Booking.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MovieReservation.DTO
{
    public class Booking
    {
        public int BookingID { get; set; }
        public string BookingName { get; set; }
        public decimal BookingAmount { get; set; }
        public int BookingStatus { get; set; }
        public List<int> BookingSeats { get; set; }
        public DateTime BookingTimeStamp { get; set; }
    }
}
